pyaedt/hfss.py

```diff
@@ -6245,11 +6276,15 @@ def wave_port(
             sheet_name = self.modeler.convert_to_selections(signal, True)[0]
             if isinstance(sheet_name, int):
                 try:
+                    # NOte: if isinstance(sheet_name, cad.elements3d.FacePrimitive) then
```

Comment:
@dcrawforAtAnsys here I think you could use the new method and then do:
face = self.modeler.get_face_by_id(id)
object = [obj for obj in self.modeler.object_list if face in obj.faces][0]